### DISCUSSION TITLE
Problem z sensorami?

### DISCUSSION THREAD

**Discussion by luke94x** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#159] [Q&A] [answered]
Cześć,

Tak jak na zrzucie, mam problem z sensorami. 3 Encje nie sa dostepne, czy moglibyscie podpowiedziec co moze byc problemem? 
Dodatkowo wrzucilem sobie do pliku konfiguracyjnego ten skrypt, ktory powinien pokazac ile energii zostalo prosumentowi, ale nigdzie tego nie widze. Przyznam, ze dopiero raczkuje w HA. Będę wdzięczny za wszystkie odpowiedzi z podpowiedzią. 

![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/59284061/7e1fab1c-4547-4cfb-83f3-87c810790aa3)

**Comment by PiotrMachowski** [comment, marked as answer]
A jaką masz konfigurację integracji? Możesz ją zobaczyć wchodząc w opcje (ale nie dawaj "Zapisz" przy zamykaniu okna)

**Reply by luke94x** [reply]
![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/59284061/2a349ca7-bdbc-4c03-9a88-9cde1731cf84)


Ok wychodzi na to, że ustawienie dla tych 3 encji nie zostało wybrane, czasem wystarczy mała podpowiedź żeby człowiek ruszył głową.
W takim razie co mogę zrobić ze skryptem do liczenia pozostalej energii jako prosument?

**Reply by PiotrMachowski** [reply]
> W takim razie co mogę zrobić ze skryptem do liczenia pozostalej energii jako prosument?

Czy walidacja yaml przechodzi? Te encje nie pojawią się na ekranie integracji, musisz je znaleźć ręcznie w narzędziach deweloperskich

**Reply by luke94x** [reply]
![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/59284061/3d9ed90d-c845-4597-a312-a602f281772c)

tak, jest widoczna w developer tools.  Pytanie jaki jest następny krok

**Reply by PiotrMachowski** [reply]
Ok, a zmieniłeś id encji w skopiowanej konfiguracji?

**Reply by luke94x** [reply]
![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/59284061/b1a4f443-af57-4551-9554-2e240b9be94b)

nic nie zmieniałem, rozumiem, że tu gdzie podkreślone należy wrzucic nr mojego licznika zamiast 123?

**Reply by PiotrMachowski** [reply]
Tak, w te miejsca musisz wrzucić id odpowiednich encji

**Reply by luke94x** [reply]
Tak teraz podaje wartość 3664, rozumiem, że tyle prądu zostało mi do wykorzystania?
Czy można to teraz dorzucić do integracji na dashboardzie?

**Reply by PiotrMachowski** [reply]
> Tak teraz podaje wartość 3664, rozumiem, że tyle prądu zostało mi do wykorzystania?

Potencjalnie, jeśli masz dobrą konfigurację.

> Czy można to teraz dorzucić do integracji na dashboardzie?

Tak, to normalna encja

**Reply by luke94x** [reply]
![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/59284061/c3544f34-28a4-48a3-a7d7-59db721e165a)

Raczej dobra, wrzucilem id tych sensorow, które wskazałeś. 

Dzięki wielkie, zrobione!!! Będzie kawka :)
